internal/utils/edgedb/queries/insertPolicyConfig.edgeql

```diff
@@ -0,0 +1,21 @@
+insert PolicyConfig {
+    name := <str>$0,
+    policyState := <str>$1,
+    policyName := (
+        select Policy
+        filter .name = <str>$2
+        limit 1
+    )
+}
+unless conflict on .name
+else (
+    update PolicyConfig
+    set {
+        policyState := <str>$1,
+        policyName := (
```

Comment:
If the Policy is updated after the PolicyConfig or PolicyException is created, this is probably never updated, right?

Reply:
Since it is a link, it is always up to date.